### PR TITLE
Publish design team blogpost

### PR DESCRIPTION
**Pull request by kwannoel**

### CODE REVIEW

File: blog/2021-03-09-Introduction-to-UX-research-Part-1.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+# **The UX Research process**
+
+1. **Ask the right questions**: Define research scope  
+    - **What do we want to research on, and how do we obtain the relevant information?**
+    - **Who are our target users, first and foremost?**
+
+    Knowledge of the users’ mindsets and differing workflows will help to determine ways to **meet their needs and address pain points.** Other than the target group, also consider studying other teams and professionals that work alongside that target group and talking to other professionals who know a lot about the target group. 
+
+2. **Critically observe**: Collect data ****
```
Comment (jhrcek) [inline blog/2021-03-09-Introduction-to-UX-research-Part-1.md]: Are these 3 dots useful for anything format-wise or just leftover?

```suggestion
2. **Critically observe**: Collect data
```

File: blog/2021-03-09-Introduction-to-UX-research-Part-2.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+![](/images/blogposts/ux_research_whiteboard.png)
+
+# **Behind the Scenes**
+
+As mentioned in Part 1 of this blog post series, during my internship, I helped with creating a questionnaire on [UserTesting](https://app.usertesting.com/users/sign_in), an online platform which allows **users from all over the world to participate in user research studies anonymously.**  Via UserTesting, we recruited a range of professionals in the pharmaceutical industry. One of the key participants, whom we shortlisted as a potential candidate for further interviewing, was in a pharmacovigilance role, and he provided a lot of useful insights via the questionnaire. The team also approached business, IT and data professionals both within and outside Holmusk for industry-specific details to get a bird’s eye view of what pharma companies were interested in.
```
Comment (jhrcek) [inline blog/2021-03-09-Introduction-to-UX-research-Part-2.md]: nit: Maybe also link to part one from here..

```suggestion
As mentioned in [Part 1]((2021-03-09-Introduction-to-UX-research-Part-1.html) of this blog post series, during my internship, I helped with creating a questionnaire on [UserTesting](https://app.usertesting.com/users/sign_in), an online platform which allows **users from all over the world to participate in user research studies anonymously.**  Via UserTesting, we recruited a range of professionals in the pharmaceutical industry. One of the key participants, whom we shortlisted as a potential candidate for further interviewing, was in a pharmacovigilance role, and he provided a lot of useful insights via the questionnaire. The team also approached business, IT and data professionals both within and outside Holmusk for industry-specific details to get a bird’s eye view of what pharma companies were interested in.
```

**Review by jhrcek — APPROVED** [review]
Looks good. No typos/language issues found a minor markdown nitpic.

**Comment by kwannoel** [comment]
Thanks @jhrcek for reviewing!

**Pull request merged**